label_studio/projects/api.py

```diff
@@ -195,11 +195,14 @@ def delete(self, request, *args, **kwargs):
     @swagger_auto_schema(tags=['Projects'], request_body=ProjectSerializer)
     def patch(self, request, *args, **kwargs):
         project = self.get_object()
-        label_config = self.request.query_params.get('label_config')
+        label_config = self.request.data.get('label_config')
 
         # config changes can break view, so we need to reset them
-        if parse_config(label_config) != parse_config(project.label_config):
-            View.objects.filter(project=project).all().delete()
+        try:
+            if label_config and parse_config(label_config) != parse_config(project.label_config):
```

Comment:
there is another function in this file called `config_essential_data_has_changed` - it's better to modify and reuse it here